client/app/application.coffee

```diff
@@ -25,8 +27,13 @@ class App extends Application
     - layout: the application layout view, rendered.
     ###
     initialize: ->
+        steps = require './steps/config'
         @on 'start', (options) =>
-            @router = new Router app: @
+
+            @onboarding = new Onboarding({}, steps)
+            @onboarding.onStepChanged (step) => @handleStepChanged(step)
```

Comment:
@gregorylegarec : couldnt you use `baconjs` to listen to data? If not could you explain it simply (with an example?)